scrapi/base/__init__.py

```diff
@@ -20,13 +21,32 @@
 logger = logging.getLogger(__name__)
 
 
+class HarvesterMeta(abc.ABCMeta):
+    def __init__(cls, name, bases, dct):
+        super(HarvesterMeta, cls).__init__(name, bases, dct)
+        if not isinstance(cls.short_name, abc.abstractproperty):
```

Comment:
This should throw an error if it isn't there, otherwise we can miss harvesters (like what happened with ucescholarship)